content/docs/user-guide/editor/entity-inspector.md

```diff
@@ -1,11 +1,28 @@
 ---
-description: ' Use the Entity Inspector to add components to entities and modify
-  their properties in Open 3D Engine. '
+linkTitle: Entity Inspector
 title: Entity Inspector
+description: Use Entity Inspector to add components to entities and modify their properties in Open 3D Engine (O3DE).
+weight: 300
 ---
 
 The **Entity Inspector** manages all the components for each entity. Select an entity in the **Entity Outliner** or the viewport to see the attached components in the **Entity Inspector**.
 
+Use the **Entity Inspector** to do the following:
```

Comment:
question: Again here there's a 'the Entity Inspector' vs 'Entity Inspector' (see page description) mismatch

Reply:
This is an example of a page that is still written in Lumberyard's style.  It needs a full update like many sub-topics in the Editor section.  Probably covered by an existing issue: https://github.com/o3de/o3de.org/issues/1490.

Reply:
Sounds great! 👍